qcelemental/util/__init__.py

```diff
@@ -1,10 +1,8 @@
 from .autodocs import auto_gen_docs_on_demand, get_base_docs
 from .gph_uno_bipartite import uno
-from .importing import parse_version, safe_version, which, which_import
+from .importing import which, which_import
```

Comment:
I think we need to keep these fns, as they're used downstream https://github.com/MolSSI/QCEngine/blob/master/qcengine/programs/rdkit.py#L8

Reply:
Fixed!